CHANGELOG.md

```diff
@@ -0,0 +1,575 @@
+# Changelog
+
+## v1.14.0-snapshot.0
+
+Summary of Changes
+------------------
+
+**Major Changes:**
+* Add per-node configuration overrides. There is a new Kubernetes resource type, CiliumNodeConfig, which allows for fine-grained configuration of Nodes based on label selectors. (#22656, @squeed)
+* Add WireGuard host2host and LB encryption (#19401, @brb)
+* cilium: completion of nat46/64 gateway (#22421, @borkmann)
+* Initial datapath support for Cilium mTLS has been added. (#21822, @jrajahalme)
+* policy: Promote Deny Policies from Beta to Stable (#22966, @nathanjsweet)
+* Support Kubernetes v1.21 new field internalTrafficPolicy=Local. (#21871, @gentoo-root)
```

Comment:
Hmm, this looks like it was generated from a wider range of PRs than intended. Many of these landed in v1.13.0 IIRC. :thinking:

Reply:
I think I found the issue, fix incoming.